### PR TITLE
Added MathJax library to close #125

### LINKED ISSUE #125: possible to use latex syntax in research notes + wiki?
I.e. so that we can write $a=\frac{3}{x}$ and get:

frac

It might be neat for writing out equations w/ units in them, so that everything ends up nice and tidy and clear ...

### PR DESCRIPTION
**Pull request by aryanbhasin**

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -16,6 +16,12 @@
     <link rel="apple-touch-icon-precomposed" href="/images/pl-255.png">
     <link rel="apple-touch-icon-precomposed apple-touch-icon" href="/images/pl-255.png">
     <link rel="shortcut icon" href="/images/pl-255.png" />
+    <script type="text/javascript" async
+      src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-MML-AM_CHTML">
+    </script>
+    <script type="text/x-mathjax-config">
+      MathJax.Hub.Config({tex2jax: {inlineMath: [['$','$'], ['\\(','\\)']]}});
```
Comment (jywarren) [inline app/views/layouts/application.html.erb]: Hello @publiclab/reviewers - does this seem OK to include on every page, and will it cause any trouble? Just trying to think broadly here -- does it run on all browsers, and are there any other failure or optimization issues we need to consider before merging this?

Thank you!

**Comment by jywarren** [comment]
This looks great. I think we should make some page of documentation of what syntax to use to do this. Perhaps on this page: https://publiclab.org/wiki/authoring-help#Advanced+formatting

**Comment by jywarren** [comment]
And also please try it in the rich text editor, at `/editor` -- in both markdown and rich text mode? What happens?

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (ananyo2012) [inline app/views/layouts/application.html.erb]: I think including this in every page is ok as it is needed in wikis, research notes and questions. Just need to test the browser compatabilites

Comment (ykl7) [inline app/views/layouts/application.html.erb]: I think MathJax is compatible across later versions of all mainstream browsers.

**Comment by ananyo2012** [comment]
Ok Jeff trying it out

**Comment by aryanbhasin** [comment]
Are you merging the PR?

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (ananyo2012) [inline app/views/layouts/application.html.erb]: Yes I have too tested it out on Firefox and chrome and it is working in both.

**Comment by ananyo2012** [comment]
@aryanbhasin Sorry to keep you waiting. We will merge it soon. I see there is a problem when we use the `^` (exponent) operator. So I am trying it out with some other mentioned libraries.

**Comment by ananyo2012** [comment]
@jywarren When testing in `/editor` the raw equation syntax is showing up for now. We can later integrate with the rich text module but that would be another issue.

**Comment by ananyo2012** [comment]
Ok the problem using exponent operator is solved by using `p` tag around the equation. It works great. I am posting few screenshots here.

**Comment by ananyo2012** [comment]
equation1
equation2

**Comment by ananyo2012** [comment]
@jywarren I think this can be merged now. The Mathjax library gives a whole set of features for using equations. After this is merged we can create a wiki or research note on its usage.

**Comment by jywarren** [comment]
Great, merging! Thanks! 🎈

**Comment by jywarren** [comment]
Can you share the strings you used to generate those sections? And post it on https://publiclab.org/wiki/authoring-help#Advanced+formatting (or make a new GCI issue for that? Thank you!

**Pull request merged**